smart-cards/src/main/java/gov/va/api/health/smartcards/Dstu2MetadataController.java

```diff
@@ -0,0 +1,119 @@
+package gov.va.api.health.smartcards;
+
+import static java.util.stream.Collectors.toList;
+
+import gov.va.api.health.dstu2.api.datatypes.ContactPoint;
+import gov.va.api.health.dstu2.api.elements.Reference;
+import gov.va.api.health.dstu2.api.resources.Conformance;
+import java.util.List;
+import java.util.stream.Stream;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Value;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.info.BuildProperties;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(
+    value = {"/dstu2/metadata"},
+    produces = {"application/json", "application/fhir+json"})
+@AllArgsConstructor(onConstructor = @__({@Autowired}))
```

Comment:
Lombok offers a more concise way to handle annotations on generated methods now: `onConstructor_ = @Autowired`